test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua

```diff
@@ -0,0 +1,111 @@
+   {rpc = "SetAppIcon", params = {syncFileName = "icon.png"}},
+   {rpc = "SetDisplayLayout", params = {displayLayout = "ONSCREEN_PRESETS"}},
+   {rpc = "SendLocation", params = {address = {countryName = "countryName"}}},
+ --{rpc = "DialNumber", params = {{number = "#3804567654*"}}},
+ --{rpc = "UpdateTurnList", params = {}}
+ --{rpc = "Speak", params = {}}, 
+ --{rpc = "SetMediaClockTimer", params = {}}
+ --{rpc = "PerformAudioPassThru", params = {}}
+ --{rpc = "PerformInteraction", params = {}},
+ --{rpc = "SubscribeButton", params = {}}, 
+ --{rpc = "UnsubscribeButton", params = {}},    
+-- {rpc = "GenericResponse", params =   {}}                   
+-- {rpc = "SystemRequest", params = {}},
+ --{rpc = "GetWayPoints", params = {}},
+-- {rpc = "SubscribeWayPoints", params = {}},
+-- {rpc = "UnsubscribeWayPoints", params = {}},
+-- ATF issue - "mandatory parameter spaceAvailable not present"
+-- {rpc = "ListFiles", params = {}},
+ --{rpc = "PutFile", params = {syncFileName ="icon.png", fileType ="GRAPHIC_PNG", persistentFile =false, systemFile = false, offset =0, length =11600}},
+ --{rpc = "DeleteFile", params = {syncFileName = "test.png"}},
+}
+
+--[[ General Precondition before ATF start ]]
+--Copy attached json file in SDL_bin folder.
+--Json contains application 123abc with NULL policies
+commonSteps:DeleteLogsFileAndPolicyTable()
+--[[ Preconditions ]]
+Test = require('connecttest')
+function Test:OpenNewSession()
+commonFunctions:userPrint(33, "================= Precondition ==================")
+  self.mobileSession2 = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession2:StartService(7)
+end
+--[[Test_1]]
+function Test:RAI_InNewSession_WithNULL_Polices()
+commonFunctions:userPrint(33, "================= Test_Case1 ====================")
+  local registerAppInterfaceParams =
+  {
+    syncMsgVersion =
+    {
+      majorVersion = 3,
+      minorVersion = 0
+    },
+    appName = "Media Application",
+    isMediaApplication = true,
+    languageDesired = 'EN-US',
+    hmiDisplayLanguageDesired = 'EN-US',
+    appHMIType = {"NAVIGATION"},
+    appID = "123abc",
+    deviceInfo =
+    {
+      os = "Android",
+      carrier = "Megafon",
+      firmwareRev = "Name: Linux, Version: 3.4.0-perf",
+      osVersion = "4.4.2",
+    }
+  }
+local corId = self.mobileSession2:SendRPC("RegisterAppInterface", registerAppInterfaceParams)
+  self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+end
+
+--[[Test_2]]
+function Test:CheckRPCs_ForApp_withPolicyNull()
+  commonFunctions:userPrint(33, "================= Test_Case2 ====================")
+ for i = 1 , #RPC do
+  local corId = self.mobileSession2:SendRPC(RPC[i].rpc, RPC[i].params) 
+  self.mobileSession2:ExpectResponse(corId, {success = false, resultCode = "DISALLOWED" })
+end
+end
+--[[ Postconditions ]]
```

Comment:
Please organize section like shown in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/27/files